tools/src/test/python/dlpx/virtualization/_internal/commands/test_build.py

```diff
@@ -66,9 +66,9 @@ def test_build_success(mock_relative_path, mock_install_deps,
         'dlpx.virtualization._internal.plugin_dependency_util.install_deps')
     @mock.patch('os.path.isabs', return_value=False)
     def test_build_success_non_default_output_file(
-        mock_relative_path, mock_install_deps, mock_generate_python,
-        mock_import_plugin, plugin_config_file, artifact_file,
-        artifact_content, codegen_gen_py_inputs):
+            mock_relative_path, mock_install_deps, mock_generate_python,
```

Comment:
did formatting change somehow? seems weird that this was edited?

Reply:
Ah yes, these lines were failing linting (`python -m flake8 tools/src/test/python/ --max-line-length 88`) in `tools` so I decided to fix it.